test/integration/clone-cleanup.js

```diff
@@ -60,17 +54,19 @@ test('clone-cleanup', t => {
             break;
 
         case 3:
-            // The second batch of clones has been created and the main thread has ended
-            verifyCounts(10, 0);
+            // The second batch of clones has been created and the main thread is about to end
+            verifyCounts(10, 1);
```

Comment:
Previously, case 3 ran after the "main" thread of the project had finished, so the expected thread count was zero. Now, case 3 runs as the result of the last `say` block in that script, so the main thread is still running => one instead of zero.

Case 4 previously ran after the main thread finished, and still does, so it stays at zero.